census/static/datepicker/js/datepicker.js

```diff
@@ -30,10 +63,39 @@ $(function () {
             let search_string = getQueryParams('search');
             if (search_string !== false) {
                 data['search'] = search_string;
-                console.log(data['search']);
             }
             redirectUrl(data);
         },
         defaultDate: getSelectedDate(),
     });
 });
+
+function highlightMonthEvents(month, year) {
+    /*
+    Given a month and year, highlight days with events
+    Note: Month provided in the argument is 1-indexed
+    and not 0-indexed as is normally used in JavaScript.
+     */
+    $.ajax({
+        url: '/',
+        method: 'GET',
+        data: {
+            isMonthly: true,
+            month: month,
+            year: year,
+        },
+        headers: {
+            credentials: 'include'
+        },
+        dataType: 'json',
+        success: function (data) {
+            let eventDates = data.dates;
+
+            $('#datepicker').find(`[data-month='`+(month-1)+`']`)
+                .find("a").filter(function () {
+                return eventDates.indexOf($(this).text()) > -1;
+            }).addClass('ui-datepicker-active-events');
+        }
+    });
+}
```

Comment:
Seems to highlight events in the future but not in the past, which is probably ok, but we display the ones in the past on the home page, maybe we should not?

Reply:
It should highlight all events in the month, including past events. I tested it again now and it seems to be working correctly. Is the past event showing up on the list in your homepage? Both calls essentially use the same logic so the behavior shouldn't differ.

Reply:
Here is a screen shot.  Some of the past events are highlighted and some are not.  I think that the past events should not be shown on the home page in any event.  We could leave that for a later enhancement. 
![Screen Shot 2019-12-22 at 6 11 44 PM](https://user-images.githubusercontent.com/6759416/71331377-c25a5780-24e6-11ea-9744-d3bbc4b58292.png)

Reply:
If you want me to gather other information, let me know.

Reply:
Thanks @mikeubell. You were right, there was an issue in highlighting single digit dates. I've fixed it now. I'll merge it.